pkg/reconciler/openshift/tektonresult/extension.go

```diff
@@ -64,11 +66,17 @@ func OpenShiftExtension(ctx context.Context) common.Extension {
 
 	}
 
+	logsRBACManifest, err := getloggingRBACManifest()
+	if err != nil {
+		logger.Fatal("Failed to fetch logs RBAC manifest")
```

Comment:
also include the `err` on the logger

Reply:
Done for 2 other logger instances as well